index.js

```diff
@@ -544,14 +437,51 @@ instance.prototype.init_actions = function(system) {
 		'app_restart': {
 			label: 'Restart companion'
 		}
-	});
+	};
+
+	for (var act of self.BUTTON_ACTIONS) {
```

Comment:
@istnv I would rather you declare these fields as constants above the actions, such as `FIELD_PAGE` and `FIELD_BUTTON` and include them in the options arrays in the actions that way.  I need to be able to read these modules as easily as possible and this loop method ain't cutting it unfortunately.

Reply:
I started by adding 'This' to a couple of actions, then realized how many more it would apply to.
Since all of the BUTTON_ACTIONS had the same dropdown options, the action and label being the unique part I was trying to consolidate adding the same options to all of them without missing one.
I can unwind the loop if that is easier to read.

Looking over the code again:
All of the actions that use CHOICES_BANKS can use the 'This' option so I'll add that to the earlier definition.
The action 'set_page' would be the only one that may not use the 'This' option, although it may be useful to set a different SURFACE to This page. Setting 'Current Surface' to 'This Page' would be a no-op.

Reply:
'This' as the button or page being acted on or referred to. Not to be confused with `this`.